generated/provider_dependencies.json

```diff
@@ -421,7 +421,7 @@
     "deps": [
       "apache-airflow>=2.8.0",
       "more-itertools>=9.0.0",
-      "sqlparse>=0.4.2"
+      "sqlparse>=0.5.1"
```

Comment:
BTw. @eladkal - don't you think it should come as part of #41916  -> it would be put in context "we bump it because we are using this new feature"?

Reply:
I see now that https://github.com/apache/airflow/pull/41916/commits/5f94f337b1ae48c9a55f5b0d348d653088c90bdc was added so we can close this one. Had problems with pushing commit to the author fork

Reply:
Yeah. I like it better too  - this one would be linked via comment but not "straightforward" really. If I search with git blame for example, it would show  me much more reasoning in the single commit that I'd find, without looking for the "related" one.

Reply:
That's true, good call @potiuk